katdal/lazy_indexer.py

```diff
@@ -341,11 +341,12 @@ def __init__(self, dataset, keep=(), transforms=None):
         except NotImplementedError:
             # Dask does not like multiple boolean indices: go one dim at a time
             for dim, keep_per_dim in enumerate(keep):
+                # Use da.take(dataset, index, axis=dim) once dask is required
```

Comment:
Presumably dask is required for DaskLazyIndexer - so you could `import dask` inside `__init__`.

Reply:
Not required but implied... I've gone this route with a note to change it eventually.